core/tsconfig.json

```diff
@@ -1,11 +1,8 @@
 {
   "compilerOptions": {
     "target": "esnext",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowArbitraryExtensions": true,
```

Comment:
Necessary for TypeScript to repesct the `.d.json.ts` file that `next-intl` is generating for strict `Messages` typing.